Python/Cloud SQL Query Notebook.ipynb

```diff
@@ -82,9 +82,20 @@
    },
    "outputs": [
     {
-     "output_type": "stream",
      "name": "stdout",
-     "text": "CPU times: user 2 µs, sys: 1 µs, total: 3 µs\nWall time: 3.81 µs\nRequirement already satisfied: pyarrow in /Users/tmhoangtus.ibm.com/anaconda3/lib/python3.6/site-packages (0.16.0)\nRequirement already satisfied: six>=1.0.0 in /Users/tmhoangtus.ibm.com/anaconda3/lib/python3.6/site-packages (from pyarrow) (1.15.0)\nRequirement already satisfied: numpy>=1.14 in /Users/tmhoangtus.ibm.com/anaconda3/lib/python3.6/site-packages (from pyarrow) (1.19.0)\n\u001b[33mWARNING: You are using pip version 20.0.2; however, version 20.1.1 is available.\nYou should consider upgrading via the '/Users/tmhoangtus.ibm.com/anaconda3/bin/python -m pip install --upgrade pip' command.\u001b[0m\nRequirement already satisfied: sqlparse in /Users/tmhoangtus.ibm.com/anaconda3/lib/python3.6/site-packages (0.3.1)\n\u001b[33mWARNING: You are using pip version 20.0.2; however, version 20.1.1 is available.\nYou should consider upgrading via the '/Users/tmhoangtus.ibm.com/anaconda3/bin/python -m pip install --upgrade pip' command.\u001b[0m\n"
+     "output_type": "stream",
+     "text": [
+      "CPU times: user 2 µs, sys: 1 µs, total: 3 µs\n",
+      "Wall time: 3.81 µs\n",
+      "Requirement already satisfied: pyarrow in /Users/tmhoangtus.ibm.com/anaconda3/lib/python3.6/site-packages (0.16.0)\n",
+      "Requirement already satisfied: six>=1.0.0 in /Users/tmhoangtus.ibm.com/anaconda3/lib/python3.6/site-packages (from pyarrow) (1.15.0)\n",
+      "Requirement already satisfied: numpy>=1.14 in /Users/tmhoangtus.ibm.com/anaconda3/lib/python3.6/site-packages (from pyarrow) (1.19.0)\n",
+      "\u001b[33mWARNING: You are using pip version 20.0.2; however, version 20.1.1 is available.\n",
+      "You should consider upgrading via the '/Users/tmhoangtus.ibm.com/anaconda3/bin/python -m pip install --upgrade pip' command.\u001b[0m\n",
+      "Requirement already satisfied: sqlparse in /Users/tmhoangtus.ibm.com/anaconda3/lib/python3.6/site-packages (0.3.1)\n",
+      "\u001b[33mWARNING: You are using pip version 20.0.2; however, version 20.1.1 is available.\n",
+      "You should consider upgrading via the '/Users/tmhoangtus.ibm.com/anaconda3/bin/python -m pip install --upgrade pip' command.\u001b[0m\n"
+     ]
```

Comment:
JFYI: The vast majority of changes in this file are those reformats of newline and sorting keys in alphabetical order.